content/docs/user-guide/components/reference/atom/sky-atmosphere.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Sky Atmosphere Component
+linktitle: Sky Atmosphere
+description: 'Open 3D Engine (O3DE) Sky Atmosphere component reference.'
+toc: true
+---
+
+The **Sky Atmosphere** component provides physically-based sky and atmosphere rendering capabilities.
+
+The sky is rendered by sampling points on rays that start at the camera and intersect with the atmosphere.  Because sampling many rays every frame is an expensive operation, luminence and volume look up tables are used to perform fewer calculations.  
+
+![Sky atmosphere in desert mountain scene](/images/user-guide/components/reference/atom/sky-atmosphere/desert-day.jpg)
+
+## Provider
+
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+
+## Properties
+
+![sky-atmosphere-component-properties](/images/user-guide/components/reference/atom/sky-atmosphere/sky-atmosphere-component-properties.png)
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Origin** | The origin to use for the atmosphere.  When `Ground at world origin` (default) is selected, the bottom of the atmosphere - the ground - will be at the world origin, or `0,0,0`.  If `Ground at local origin` is used, then the bottom of the atmosphere will be at the world position of the Transform Component for the entity the Sky Atmosphere component is on.  Using `Ground at local origin` gives you an easy way to control the position of the bottom of the atmosphere.  One practical use for `Ground at local origin` is to move the ground position down so you have a more gradual gradient transition from surface to sky.  Try using `Ground at local origin` and moving the entity transform down by setting the `Z` value to `-100` to get a smoother transition. The `Planet center at local origin` is makes the center of the planet for the atmosphere at the world position of the Transform component for the entity the Sky Atmosphere component is on.  The `Planet center at local origin` is useful when you want to have atmosphere for planets. | `Ground at world origin`,`Ground at local origin`, `Planet center at local origin`| `Ground at world origin` |
```

Comment:
```suggestion
| **Origin** | The origin to use for the atmosphere.  When `Ground at world origin` (default) is selected, the bottom of the atmosphere - the ground - will be at the world origin, or `0,0,0`.  If `Ground at local origin` is used, then the bottom of the atmosphere will be at the world position of the Transform Component for the entity the Sky Atmosphere component is on.  Using `Ground at local origin` gives you an easy way to control the position of the bottom of the atmosphere.  One practical use for `Ground at local origin` is to move the ground position down so you have a more gradual gradient transition from surface to sky.  Try using `Ground at local origin` and moving the entity transform down by setting the `Z` value to `-100` to get a smoother transition. The `Planet center at local origin` makes the center of the planet for the atmosphere at the world position of the Transform component for the entity the Sky Atmosphere component is on.  The `Planet center at local origin` is useful when you want to have atmosphere for planets. | `Ground at world origin`,`Ground at local origin`, `Planet center at local origin`| `Ground at world origin` |
```

Reply:
there was a small typo on the sentence `The `Planet center at local origin` is makes the center of...`